src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -1913,6 +1926,7 @@ void ElectricEIRChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, b
 
     // If not air or evap cooled then set to the condenser node that is attached to a cooling tower
     Real64 condInletTemp = state.dataLoopNodes->Node(this->CondInletNodeNum).Temp;
+    this->CondInletTemp = condInletTemp; // needed for thermosiphon model
```

Comment:
If I revert this change in the EIR and ReformulatedEIR chillers there are no eio or results diffs in example file ASHRAE901_Hospital_STD2019_Denver.